### ISSUE TITLE
Dubbing: Target language is not getting set.

### ISSUE THREAD

**Issue by danielmb** [elevenlabs/elevenlabs-js#44]
Hello.

I am trying to trying to use the dubbing api using this module. While using dubbing part I am getting this error everytime:

```json
{
  "detail": {
    "status": "empty_target_language",
    "message": "Target language must be set in automatic dubbing mode. Either set the target language or set dubbing studio to true."
  }
}
``` 

This is the code I wrote to reproduce the error:

```js
import { ElevenLabsClient } from 'elevenlabs';
import dotenv from 'dotenv';
import fs from 'fs';

dotenv.config();

const elevenlabs = new ElevenLabsClient({
  apiKey: process.env.ELEVENLABS_API_KEY,
});

const file = fs.createReadStream('test.mp4');

const res = await elevenlabs.dubbing
  .dubAVideoOrAnAudioFile({
    file,
    target_lang: 'en',
    watermark: true,
    mode: 'automatic',
  })

console.log(res);
```

I am running node v20.13.1

**Comment by chandrashekhar07** [comment]
same thing happening on node v20.11.1 and     "elevenlabs": "^0.6.0"

**Comment by jaiswar203** [comment]
Facing the same issue, how are you guys doing workaround? using API Service, if so can you send me your implementation for rerference @danielmb @chandrashekhar07

**Comment by chandrashekhar07** [comment]
But it works through API calls,

**Comment by dsinghvi** [comment]
@chandrashekhar07 this should be fixed in version `0.8.2`!
